pages/signInSignUp.js

```diff
@@ -5,7 +5,7 @@ import LoginForm from '../components/loginform';
 import '../static/styles/main.scss'
 
 const SignInSignUp = () => (
-  <div className="main-container">
+  <div className="main-container" style={{backgroundImage: `url(${"/static/images/abc_blackboard.jpg"})`, backgroundSize:'100% 80%', backgroundRepeat: 'no-repeat'}}>
```

Comment:
could we put the styles in the _login.scss file and then add an additional className to this div?

something like:

`<div className="main-container login-container">`